### DISCUSSION TITLE
水星記MV中的機器人

### DISCUSSION THREAD

**Discussion by wu51000** [1101-GS3073-C/Python-Community-GS3073-C#748] [General]
不知道有多少人看過水星記的MV
因為角色切換很快所以我聽了好幾遍但還是沒有全懂
突然想到 就認真的去查了
—
看完文章後我不斷的思考
用虛擬實境讓人們看已經不存在的事物
對失憶的人們來說究竟是福音
還是一種傷痛呢
那是一段很傷心很傷心的故事
在幻境中什麼都有 關於不在的人們 曾經的往事
在幻境中有多快樂 回歸現實的時候就有多痛苦
—
若今天失憶的人是你，你會選擇承擔這份痛苦而在所不惜的去回憶嗎
又或是寧願不回想起往事，繼續著原先的生活呢
歡迎跟我分享你的選擇！
https://vocus.cc/article/5eeb6b46fd897800011adb2f

**Comment by yjpxi** [comment]
我覺得使用虛擬實境讓人們觀看已經不存在的事物，對於失憶者來說可能是一把雙刃劍。它可以是一種福音，因為它有助於恢復失去的記憶，讓人重新體驗過去的快樂時光；然而，它同時也可能帶來痛苦，因為這種虛擬的快樂與現實的孤寂形成了強烈的對比，回到現實後可能會加劇情感上的失落感。
